include/bluetooth/bluetooth.h

```diff
@@ -343,6 +343,10 @@ struct bt_le_adv_param {
  *  @param sd_len Number of elements in sd
  *
  *  @return Zero on success or (negative) error code otherwise.
+ *  @return -EISCONN when connectable advertising is requested and there
+ *  is already maximum number of connections established. This error
```

Comment:
indentation, align with the start of the description and capital 'W' in 'When' .